pset-functions.js

```diff
@@ -175,6 +216,27 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+
+function isPalindrome(word) {
```

Comment:
@jimernest90 interesting approach, here you are using recursive programming by having the function call itself.

Take a look at an alternative approach [here](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-functions.js#L246) that just compares the original word to the sorted & reversed version of the word determines if they are the same